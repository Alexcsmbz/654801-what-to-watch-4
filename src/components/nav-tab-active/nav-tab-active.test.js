import NavTabActive from './nav-tab-active.jsx';

const tab = {
  name: `Overview`,
};

describe(`NavTabActive snapshot test`, () => {
  it(`NavTabActive should render nav tab active`, () => {
    const tree = renderer.create(
        <NavTabActive
          onClick={() => {}}
          tab={tab}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
