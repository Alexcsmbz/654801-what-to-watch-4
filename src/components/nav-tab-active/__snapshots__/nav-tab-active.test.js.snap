// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavTabActive snapshot test NavTabActive should render nav tab active 1`] = `
<li
  className="movie-nav__item movie-nav__item--active "
  onClick={[Function]}
>
  <div
    className="movie-nav__link"
  >
    Overview
  </div>
</li>
`;
